### PR TITLE
Add missing step from configuring tz doc

### PR DESCRIPTION
**Pull request by saifabusaleh**
Add restart the engine step that was missing
to configuring tz document.

This pull request needs review by: @ahadas @dcdacosta @sandrobonazzola

### CODE REVIEW

File: source/documentation/virtual_machine_management_guide/topics/Configuring_Time_Zones.adoc [hunk 1]
```diff
@@ -19,3 +19,5 @@ For example, `10-timezone.properties` overrides the default file, `00-timezone.p
 +
 General:: Time zones used for non-Windows operating system types, must follow the standard time zone format for example, `Etc/GMT` or `Asia/Jerusalem`.
 Windows:: Time zones specifically supported on link:https://docs.microsoft.com/en-us/previous-versions/windows/embedded/ms912391(v=winembedded.11)?redirectedfrom=MSDN[Windows] for example, `GMT Standard Time` or `Israel Standard Time`.
++
+. Restart the engine.
```
Comment (ahadas) [inline source/documentation/virtual_machine_management_guide/topics/Configuring_Time_Zones.adoc:23]: engine -> `ovirt-engine` service

File: source/documentation/virtual_machine_management_guide/topics/Configuring_Time_Zones.adoc [hunk 2]
```diff
@@ -19,3 +19,5 @@ For example, `10-timezone.properties` overrides the default file, `00-timezone.p
 +
 General:: Time zones used for non-Windows operating system types, must follow the standard time zone format for example, `Etc/GMT` or `Asia/Jerusalem`.
 Windows:: Time zones specifically supported on link:https://docs.microsoft.com/en-us/previous-versions/windows/embedded/ms912391(v=winembedded.11)?redirectedfrom=MSDN[Windows] for example, `GMT Standard Time` or `Israel Standard Time`.
++
```
Comment (stoobie) [inline source/documentation/virtual_machine_management_guide/topics/Configuring_Time_Zones.adoc:22]: This line is not necessary.

File: source/documentation/virtual_machine_management_guide/topics/Configuring_Time_Zones.adoc [hunk 3]
```diff
@@ -19,3 +19,5 @@ For example, `10-timezone.properties` overrides the default file, `00-timezone.p
...
+. Restart `ovirt-engine` service.
```
Comment (stoobie) [inline source/documentation/virtual_machine_management_guide/topics/Configuring_Time_Zones.adoc:23]:
```suggestion
. Restart the `ovirt-engine` service:
+
[options="nowrap" subs="normal"]
----
# systemctl restart ovirt-engine
----
```

**Review by stoobie — CHANGES_REQUESTED** [review]
@saifabusaleh 
Looks good! Minor suggestions.

Also, as long as you're in the file, could you delete the `+` in line 17? It's not necessary.

File: source/documentation/virtual_machine_management_guide/topics/Configuring_Time_Zones.adoc (hunk 2 above)
Comment (saifabusaleh) [reply source/documentation/virtual_machine_management_guide/topics/Configuring_Time_Zones.adoc:22]: I think its needed with the suggested change

Comment (stoobie) [reply source/documentation/virtual_machine_management_guide/topics/Configuring_Time_Zones.adoc:22]: Ah, yes, you're right. ;D

**Review by ahadas — APPROVED** [review]

**Review by stoobie — APPROVED** [review]
LGTM

**Comment by sandrobonazzola** [comment]
Merging based on above approvals

**Pull request merged**